core/src/test/java/org/frankframework/jdbc/StoredProcedureQuerySenderTest.java

```diff
@@ -404,14 +420,14 @@ public void testStoredProcedureClobOutputParameter() throws Exception {
 		assertEquals(value, result.getResult().asString());
 	}
 
-	@Test
-	public void testStoredProcedureBlobOutputParameterNullValue() throws Exception {
+	@DatabaseTest
+	public void testStoredProcedureBlobOutputParameterNullValue(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
 
-		assumeThat("H2, PSQL not supported for this test case", productKey, is(not(oneOf("H2", "PostgreSQL"))));
+		assumeFalse(dataSourceName.contains("H2") || dataSourceName.contains("PostgreSQL"), "H2, PSQL, DB2 not supported for this test case");
```

Comment:
Please check everywhere in this class where DB2 is mentioned in the text, but not in the logic of the test. If so, correct it.